Changes on Synthetics documentation (#7637)
DataDog/documentation: content/en/synthetics/browser_tests/actions.md

#### Test your UI with custom JavaScript

Test a custom assertion on the active page using your own JavaScript scripts. JavaScript assertions support both synchronous and asynchronous code. 

The JavaScript assertion function comes with two parameters. You also need to make sure to include a `return` statement in your JavaScript snippet.

* `vars`: String containing your browser test [variables][5]. Use `vars.<YOUR_VARIABLE>` to refer to your browser test variable in your JavaScript snippet. Example: If your browser test contains a `USERNAME` variable, you can call it in your JavaScript snippet using `vars.USERNAME`.

* `element`: Locator of the element on the page. This element can be selected and updated with the `Select` and `Update` target element buttons of the step. The selected element automatically leverages Datadog browser test multi locating algorithm.

{{< img src="synthetics/browser_tests/js_assertion.mp4" alt="Browser Test JavaScript Assertion" video="true" width="100%">}}

> * The `return` statement needs to reflect the condition the assertion needs to meet for your browser test step to be successful. You can return any type, but the value is automatically casted as a boolean.

Reviewer: ```suggestion
* The `return` statement should return a value indicating your browser test step was successful. Any type can be returned, but the value is automatically casted as a boolean.
```

> Since JavaScript assertions run in the context of the active page, it also means these steps can access all the objects defined in the active page (libraries, builtins, global variables, ...). If you need to load external libraries, you can wrap their loading in a promise like below:

Reviewer: ```suggestion
Since JavaScript assertions run in the context of the active page, these steps can access all the objects defined in the active page (libraries, built-ins, global variables, etc.). To load external libraries, use a promise, for example:
```

{{< img src="synthetics/browser_tests/js_assertion.mp4" alt="Browser Test JavaScript Assertion" video="true" width="100%">}}

> * The `return` statement needs to reflect the condition the assertion needs to meet for your browser test step to be successful. Any type can be returned, but the value is automatically casted as a boolean.

Reviewer: ```suggestion
* The `return` statement should reflect the condition the assertion needs to meet for your browser test step to be successful. Any type can be returned, but the value is automatically casted as a boolean.
```

##### JavaScript

> Generate custom variables using your own JavaScript scripts. JavaScript steps support both synchronous and asynchronous code.

Reviewer: ```suggestion
Generate custom variables using your own JavaScript code. JavaScript steps support both synchronous and asynchronous code.
```

> This JavaScript function comes with two parameters. You also need to make sure to include a `return` statement in your JavaScript snippet.

Reviewer: ```suggestion
The JavaScript assertion function comes with the following parameters and requires a return statement.
```

> * `vars`: String containing your browser test [variables][5]. Use `vars.<YOUR_VARIABLE>` to refer to your browser test variable in your JavaScript snippet. Example: If your browser test contains a `USERNAME` variable, you can call it in your JavaScript snippet using `vars.USERNAME`.

Reviewer: ```suggestion
* `vars`: A string containing your browser test [variables][5]. Use `vars.<YOUR_VARIABLE>` to refer to a browser test variable in your JavaScript snippet. For example, if your browser test contains a `USERNAME` variable, call it in your JavaScript snippet using `vars.USERNAME`.
```

> * `element`: Locator of the element on the page. This element can be selected and updated with the `Select` and `Update` target element buttons of the step. The selected element automatically leverages Datadog browser test multi locating algorithm.

Reviewer: ```suggestion
* `element`: The locator of the element on the page. To set this up, use the **Select** and **Update** target element buttons. The selected element automatically leverages Datadog's browser test multi-locating algorithm.
```

{{< img src="synthetics/browser_tests/js_variable.mp4" alt="Browser Test JavaScript Variable" video="true" width="100%">}}

> * The `return` statement needs to reflect the condition the assertion needs to meet for your browser test step to be successful. You can return any type, but the value is automatically casted as a string.

Reviewer: ```suggestion
* The `return` statement should return a value indicating your browser test step was successful. Any type can be returned, but the value is automatically casted as a boolean.
```

> By default the step is being performed on the active page, which means it can access all the objects defined at the active page level (libraries, builtins, global variables, ...). If you need to load external libraries, you can wrap their loading in a promise like below:

Reviewer: ```suggestion
Since JavaScript assertions run in the context of the active page, these steps can access all the objects defined in the active page (libraries, built-ins, global variables, etc.). To load external libraries, use a promise, for example:
```
